lib/rbac/roles.rb

```diff
@@ -2,6 +2,8 @@ module RBAC
   class Roles
     attr_reader :roles
 
+    APPROVER_ROLE_PREFIX = 'approval-group-'.freeze
```

Comment:
Should the constant be added here? Didn't you move `lib/rbac` to `manageiq-api-common`?